go/test/endtoend/vtgate/lookup_test.go

```diff
@@ -122,7 +92,7 @@ func TestConsistentLookup(t *testing.T) {
 	mysqlErr := err.(*mysql.SQLError)
 	assert.Equal(t, 1062, mysqlErr.Num)
 	assert.Equal(t, "23000", mysqlErr.State)
-	assert.Contains(t, mysqlErr.Message, "Duplicate entry")
+	assert.Contains(t, mysqlErr.Message, "reverted partial DML execution")
```

Comment:
this is masking duplicate error. we should return the original error as well.

Reply:
It is expected to mask the duplicate error. We are using `queryserver-config-terse-errors` now and the `Duplicate entry` error message has PII information, so we don't return that.

Reply:
It is hard to like them! **_terse-errors_**